.github/workflows/on_build_do_publish_image.yml

```diff
@@ -0,0 +1,60 @@
+name: On Build Do Publish Image
+on:
+   workflow_run:
+      workflows: [On PR Open or Push Do Build]
+      types: [completed]
```

Comment:
Is the image created for every PR?

The current Jenkins images are only created when a PR is annotated with "Image Required". We don't currently have any automated mechanism in place to cleanup old quay.io/infinispan-test/server images, but I think this will be required if we create an image per PR, although I don't know what quay.io usage limits are.

Reply:
It's ok to reintroduce the label. I was thinking to do it in the next step, but better to add it from the beginning... no need to fill quay with dependabot images